src/components/StudentsTable.vue

```diff
@@ -1,16 +1,30 @@
 <template>
   <div class="students-table">
+    <div class="students-table__actions">
+      <button class="students-table__action-button" @click="addRow">Add Row</button>
+      <button
+        class="students-table__action-button"
+        :disabled="isDeleteRowsButtonDisabled"
+        @click="deleteSelectedRows"
+      >
+        {{ deleteRowsButtonMessage }}
+      </button>
+    </div>
     <AgGridVue
       class="students-table__grid ag-theme-quartz-dark"
+      rowSelection="multiple"
+      suppressRowClickSelection
       :loading="studentsStore.isFetchLoading"
       :rowData="studentsStore.students"
       :columnDefs="studentsColDefs"
       :defaultColDef="defaultStudentsColDef"
+      :getRowId="getRowId"
       @gridReady="onGridReady"
       @cellEditingStarted="onCellEditingStarted"
       @cellEditingStopped="onCellEditingStopped"
+      @selectionChanged="onSelectionChanged"
     />
```

Comment:
lets maybe add singleClickEdit on grid, it's a bit annoying that we need to click twice :D